changes/2020-07-13_detect-base64-encoded-messages/change.md

```diff
@@ -0,0 +1,88 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Detect Base64-encoded Messages
+
+## Affected Features
+
+This serves as a reference of all features that this change affects.
+
+| Feature                                                                                                                                     |
+| ------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Decrypt](https://github.com/awslabs/aws-encryption-sdk-specification/blob/623992d999db0b309d8a8adbd664f0d72feee813/client-apis/decrypt.md) |
+|                                                                                                                                             |
+
+## Affected Specifications
+
+This serves as a reference of all specification documents that this change affects.
+
+| Specification                           |
+| --------------------------------------- |
+| [Decrypt](../../client-apis/decrypt.md) |
+
+## Affected Implementations
+
+This serves as a reference for all implementations that this change affects.
+
+| Language   | Repository                                                                            |
+| ---------- | ------------------------------------------------------------------------------------- |
+| C          | [aws-encryption-sdk-c](https://github.com/aws/aws-encryption-sdk-c)                   |
+| Java       | [aws-encryption-sdk-c](https://github.com/aws/aws-encryption-sdk-java)                |
+| Javascript | [aws-encryption-sdk-javascript](https://github.com/aws/aws-encryption-sdk-javascript) |
+| Python     | [aws-encryption-sdk-javascript](https://github.com/aws/aws-encryption-sdk-python)     |
+
+## Definitions
+
+### Conventions used in this document
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Summary
+
+When a decryption attempt is made on the Base64-encoding of a valid message,
+implementations SHOULD detect this and provide a more specific error message.
+
+## Motivation
+
+The message format is a binary format, but it is a common mistake for users to attempt decryption on the Base64 encoding of this data instead.
+Because the first two bytes of the message format have a very limited set of possible values
+(currently they are in fact fixed),
+the first two bytes of the Base64 encoding of a valid message are also simple to recognize.
+
+## Drawbacks
+
+This feature technically means the version value is capped to 40 or less,
```

Comment:
40 what?

Reply:
As in literally the version must be <= 40. If we ever bumped it to 41 it would be ambiguous with the Base64 encoding of the version value "1". :) Will clarify a bit more.

Reply:
Ah, clarification would definitely be good.